### PR TITLE
Added support for React 16

### PR DESCRIPTION
**Pull request by olofd**

### CODE REVIEW

**Review by pirelenito — CHANGES_REQUESTED** [review]
Thanks for the contribution!

File: package.json [hunk 1]
```diff
@@ -1,11 +1,9 @@
 {
   "name": "lekplats",
-  "version": "0.3.1",
+  "version": "0.3.2",
```
Comment (pirelenito) [inline package.json]: The bump in the version should not be part of the PR, but instead it should be bumped after it is merged.

**Review by pirelenito — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -29,20 +27,21 @@
...
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0"
+    "react": "^15.4.2 || ^16.0.0",
+    "react-dom": "^15.4.2 || ^16.0.0"
   },
   "dependencies": {
     "classnames": "^2.2.5",
-    "html2react": "^1.0.2",
+    "html2react": "^1.1.1",
```
Comment (pirelenito) [inline package.json]: We should probably wait for your other PR to https://github.com/Deschtex/html2react/pull/27 to be merged and bumped before merging this one.

Then we will be all supporting React 16.

File: package.json (hunk 1 above)
Comment (olofd) [reply package.json]: Ok. Will remove it.

**Review by pirelenito — CHANGES_REQUESTED** [review]

**Comment by olofd** [comment]
Important. This has a dependdency to these two PR:
https://github.com/Deschtex/html2react/pull/27

and:

https://github.com/xaviervia/patch-react-proptypes-add-introspection/pull/1

When these to are out. We can merge this and bump our package.json to consume the new versions of these dependencies.

**Comment by pirelenito** [comment]
Cool! Just let me know once those are done, and we can proceed here.
